frontend/src/helper.js

```diff
@@ -154,6 +154,15 @@ export const dateFormat = () => {
   return "DD.MM.YYYY";
 };
 
+export const isDateReached = date => {
+  const today = dayjs(new Date());
+  return today.isAfter(date);
+};
+
+export const convertDatetimeToDate = datetime => {
```

Comment:
Remove this - see review comment in DatePicker.js

Reply:
done